addons/l10n_jo/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,540 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="tax_report_vat_return" model="account.report">
+        <field name="name">Tax Report Jordan</field>
+        <field name="root_report_id" ref="account.generic_tax_report"/>
+        <field name="country_id" ref="base.jo"/>
+        <field name="filter_fiscal_position" eval="True"/>
+        <field name="availability_condition">country</field>
+        <field name="column_ids">
+            <record id="tax_report_vat_return_balance" model="account.report.column">
```

Comment:
@Nadaseder

Reply:
@jco-odoo we initially did it on V15 where we added a line for the base and another line for the tax amount for each tax in the report. We can do the necessary edit to make them show on columns instead of different rows, what do you think?

Reply:
@jco-odoo 
I checked the documentation from the authorities and there's no need to add columns actually, the base amount lines are only there for the reference of the users. (Basically the report from the authority only asks for the tax amount. 

Let's keep it in rows so that people do not get confused during the declaration

Reply:
And they will be less confused with the 2 sections?